src/authz-module/components/PermissionTable.tsx

```diff
@@ -0,0 +1,52 @@
+import { Check, Close } from '@openedx/paragon/icons';
+import { Icon } from '@openedx/paragon';
+import { PermissionsResourceGrouped, Role } from '@src/types';
+import { actionsDictionary } from './RoleCard/constants';
+import ResourceTooltip from './ResourceTooltip';
+
+type PermissionTableProps = {
+  roles: Role[];
+  permissionsTable: PermissionsResourceGrouped[];
+};
+
+const PermissionTable = ({ permissionsTable, roles }: PermissionTableProps) => (
+  <table className="pgn__data-table bg-light-100">
```

Comment:
It seems odd to be applying the `DataTable` component's styles here by adding `pgn__data-table` without using the `DataTable` component.

From what I can tell all that's actually needed from there is https://github.com/openedx/paragon/blob/cce7beb990c61778616ae7d7216ff4fbc609a100/src/DataTable/index.scss#L139, which can be accomplished with

```suggestion
  <table className="w-100 bg-light-100">
```